client/components/plan-storage/README.md

```diff
@@ -0,0 +1,64 @@
+Plan Storage
+==============
+
+This component is used to display the remaining media storage limits. Using PlanStorage
+will query media storage limits for you. 
+
+#### Usage:
+
+```javascript
+import PlanStorage from 'components/plan-storage';
+
+render: function() {
+	return (
+		<PlanStorage
+			site={ this.props.site }
+			onClick={ this.onClickHandler } 
+		/>;
```

Comment:
This example is a bit strange, since obviously `render:` can't be defined as an object property outside the context of an object. The additional semi-colon here should also be removed.